src/graphics/webgpu/webgpu-bind-group.js

```diff
@@ -0,0 +1,88 @@
+import { Debug } from '../../core/debug.js';
+
+/** @typedef {import('../bind-group.js').BindGroup} BindGroup */
+/** @typedef {import('./webgpu-graphics-device.js').WebgpuGraphicsDevice} WebgpuGraphicsDevice */
+/** @typedef {import('./webgpu-texture.js').WebgpuTexture} WebgpuTexture */
+
+/**
+ * A WebGPU implementation of the BindGroup, which is a wrapper over GPUBindGroup.
+ *
+ * @ignore
+ */
+class WebgpuBindGroup {
+    /** @type {GPUBindGroup} */
+    bindGroup;
+
+    update(bindGroup) {
+
+        this.destroy();
+        const device = bindGroup.device;
+
+        /** @type {GPUBindGroupDescriptor} */
+        const descr = this.createDescriptor(device, bindGroup);
```

Comment:
again wondering if the type declaration is needed here? `createDescriptor` function in this case has a documented return type of `GPUBindGroupDescriptor` - but we must specify it again here? i hope not!

Reply:
its not needed, at least not for VSCode, but I like to document types in constructor as that makes the code easier to read .. I can easily see the types of variables, even though VSC can deduct them.